.github/workflows/main.yml

```diff
@@ -59,7 +59,8 @@ jobs:
         run: |
           (cd tests && dart pub global run stagehand:stagehand --override package-simple \
            && dart pub add sprintf \
-           && dart pub add tuple)
+           && dart pub add tuple \
+           && dart pub add vnum)
```

Comment:
odd. when I run this locally, I dont get an error.  I didn't have flutter installed.  Probably because i previously had done `dart pub add sdk`.  I am guessing that https://github.com/AmirKamali/Flutter_Vnum/issues/7 is going to take a while, as there are quite significant deps in that project on flutter, esp their tests.  IMO, add the extra deps to CI to get this merged (and maybe even check in tests/pubspec.yaml? since the list of deps is growing quickly, then we dont need to install `stagehand`), and note in the readme that wrt Dart, use of IntEnum and StrEnum depends on flutter.

Reply:
Possibly only install and run dart tests on one OS, if the deps take too long to install on all of them.

Reply:
I'm not sure if I can get install vnum by adding more dependencies. My guess is that something in dart-lang setup action is forbidding flutter, which is required for vnum.

The only solution I can think of is to use flutter setup action (which  may be more expensive) and use `flutter pub get`

Reply:
yup, try that.  we can optimise it later